src/h5web/toolbar/controls/DomainSlider/ScaledSlider.tsx

```diff
@@ -78,7 +78,7 @@ function ScaledSlider(props: Props) {
       max={SLIDER_RANGE[1]}
       value={scaledValue}
       onBeforeChange={setBeforeChangeValue}
-      onChange={handleChange}
+      onChange={(bounds) => handleChange(bounds)}
```

Comment:
We cannot yet take advantage of https://github.com/zillow/react-slider/issues/212 because of https://github.com/zillow/react-slider/issues/229